src/routes/safe/components/Transactions/TxList/ThresholdWarning.tsx

```diff
@@ -0,0 +1,12 @@
+import { ReactElement } from 'react'
+import AlertTooltipWarning from './AlertTooltipWarning'
+
+const ThresholdWarning = (): ReactElement => (
+  <AlertTooltipWarning
+    tooltip="This transaction alters the number of confirmations required to execute a transaction."
```

Comment:
When adding a new owner without changing the threshold, this tooltip message is still displayed. Is that intended?

Reply:
We could ajust the wording to say something like "This transaction _possibly_ alters..." to cover both cases

Reply:
Yes, if you add an owner, you are still calling the `addOwnerWithThreshold` method. [The textual warning was there before these changes](https://github.com/safe-global/safe-react/pull/3861/files#diff-b26291630e219ff81f6dfbc1bc81e01a516e1758e7764add36eed4188b6ff005).